autoload/nnn.vim

```diff
@@ -3,6 +3,7 @@ let s:action = ''
 let s:nnn_conf_dir = (!empty($XDG_CONFIG_HOME) ? $XDG_CONFIG_HOME : $HOME.'/.config') . '/nnn'
 " The fifo used by the persistent explorer
 let s:explorer_fifo = ""
+let t:explorer_winid = 0
```

Comment:
Probably don't need to initialize this immediately.

Reply:
Yes, it's leftover from earlier state. I'll clean it up and document the changes on a separate PR.